### ISSUE TITLE
Bottom border is not turning red from blue upon selection of an admin in “Workspace > Members”

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#20151]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1.	Go to "Settings."
2.	Click on "Profile."
3.	Click on " Personal details."
4.	Click on " Home address."
5.	Click “save” button without entering any data, notice all input field turns red with error message.
1.	Now go back and click on "Security".
2.	Click on " Close account".
3.	Click on " Close account" button.
4.	Notice the bottom border of “Default contact method” turn red as no contact has been entered.
1.	Go to "Settings."
2.	Click on " Workspaces."
3.	Click on “Members”.
4.	Select “admin”.
5.	Notice error will appear in red but bottom border hasn’t converted to red from blue color.
## Expected Result: 
Bottom border should turn red upon selection of an admin
## Actual Result: 
Bottom border doesn’t turn red upon selection of an admin

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.23-6
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/65cb8b21-1713-4339-ab30-859aec358ee5

<img width="960" alt="Screenshot (352)" src="https://github.com/Expensify/App/assets/43996225/5c76df1b-feee-4d6b-ab4f-a2a711590c8e">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @usmantariq96
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1685683898386629

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by strepanier03** [comment]
Easy to recreate with the steps so you can see a clear pattern of behavior and where we get inconsistent.

<img width="1108" alt="image" src="https://github.com/Expensify/App/assets/10925636/ea7f420b-4ffa-4a6a-9cbe-2cda87a5fd4d">

**Comment by bhrvarsani22** [comment]
Let me check...

**Comment by roryabraham** [comment]
I'm not very compelled that this is an actual bug, so I'm going to reduce the bounty to $250.

**Comment by mollfpr** [comment]
@roryabraham Sorry for the delay 🙏 and I agree it's not an actual bug.

@hungvu193 you can start to work on the PR 🚀

**Comment by mollfpr** [comment]
> There're issues with the selected proposal which were addressed in my proposal

You’re right. @roryabraham I think we could reconsider @dukenv0307 proposal.

**Comment by roryabraham** [comment]
@dukenv0307 @mollfpr looks like you're right. I admit I was moving pretty quickly here and your proposal seems superior. Since @hungvu193 hasn't started creating the PR yet, I'm going to reassign this to @dukenv0307.

My mistake

**Comment by mollfpr** [comment]
Sorry for the trouble @roryabraham @hungvu193 @dukenv0307 🙏

**Comment by strepanier03** [comment]
Bumped the [PR](https://github.com/Expensify/App/pull/20337) since it's waiting for our review.

**Comment by roryabraham** [comment]
PR merged

**Comment by strepanier03** [comment]
@trjExpensify - I am leaving for sabbatical this evening and will not be available to pay this on the 21st. Thank you for handling!

**Comment by roryabraham** [comment]
HOLDing for payment

**Comment by trjExpensify** [comment]
Okay, on the payment front. Bounty was made $250 from [here](https://github.com/Expensify/App/issues/20151#issuecomment-1579656105). 

- $500 to @dukenv0307 for the fix + #urgency bonus 
- $500 to @mollfpr for the C+ review + #urgency bonus 

(The PR was merged after +3 business days, but that was on us. C/C+ were approved and done within the timeframe, so still applying the bonus). 

Everyone happy with that?

**Comment by trjExpensify** [comment]
As for the regression test, I don't think it's required. I recently closed a bug report from Applause on this topic, so the existing regression tests in and around deleting members will catch this display issue.

**Comment by trjExpensify** [comment]
Okay, thanks for confirming. Offers sent [here](https://www.upwork.com/jobs/~01f4c2e92db90a6046).

**Comment by usmantariq96** [comment]
@trjExpensify I haven't received any offer for reporting bounty.

**Comment by trjExpensify** [comment]
Paid @mollfpr & @dukenv0307, offer sent to @usmantariq96!

**Comment by usmantariq96** [comment]
@trjExpensify offer accepted

**Comment by trjExpensify** [comment]
Perfect, thanks! Settled up with everyone, closing. :)
